01_fib/jonathanheaden/main.go

```diff
@@ -0,0 +1,34 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+)
+
+var out io.Writer = os.Stdout
+var fibs = map[int]int{}
```

Comment:
a map where each index is a sequential number from 0 is really the same data structure as an array/slice, just less efficient.

Also do you really need to store *all* the fibs, or just the last two? Can you get away with out an indexed data structure at all?

Reply:
agree. I have replaced the map with a three element array